static/js/message_live_update.js

```diff
@@ -10,6 +11,10 @@ export function rerender_messages_view() {
         }
         if (list.table_name !== undefined) {
             list.rerender_view();
+
+            for (const msg of list.data._items) {
+                update_vote_text_on_message(msg);
+            }
```

Comment:
I think this is probably the wrong abstraction for doing this update, but I assume the problem statement here is that at present, rerendering the message view won't recompute this cached `vote_text`?

Reply:
> I assume the problem statement here is that at present, rerendering the message view won't recompute this cached `vote_text`?

Yes. This function is called when toggling the setting to accordingly update all messages.





> I think this is probably the wrong abstraction for doing this update.

I don't get why this is wrong. Can you explain?